core/src/test/scala/org/apache/spark/ui/UISuite.scala

```diff
@@ -179,16 +179,15 @@ class UISuite extends SparkFunSuite {
   test("verify appUIAddress contains the scheme") {
```

Comment:
Thanks Sean! How do you manage to find so much time to help people like me?!?! You're certainly not a robot -- I do know it since I met you few times and can confirm ;-)